components/ai/code/Code.js

```diff
@@ -369,8 +370,63 @@ export default function UseCases(props) {
                 </Col>
             </Row>
 
-            {/* text connectors */}
+            {/* natural programming UX */}
             <Row className="pageContentRow integration code odd">
+                <Col xs={12}>
+                    <Container>
+                        <Row>
+                            <Col xs={12} className={styles.box}>
+                                <h2 id='natural-programming-vscode-features' className='section'>
+                                    <svg
+                                        xmlns="http://www.w3.org/2000/svg"
+                                        width="30"
+                                        height="30"
+                                        fill="currentColor"
+                                        className="bi bi-link-45deg mdButton pe-2"
+                                        viewBox="0 0 16 16"
+                                        onClick={(e) => props.getLink(e.target, 'natural-programming-vscode-features')}
```

Comment:
ditto

Reply:
Will change, 
But these are ids for the elements